### PR TITLE
Add AGOV-EN Tistina

### PR DESCRIPTION
**Pull request by FluerLee**
add 7 tistina cards

### CODE REVIEW

File: scripts/AGOV-EN/c101202090.lua
Comment (Lyris12) [inline scripts/AGOV-EN/c101202090.lua:1]: Already done: #1191

File: scripts/AGOV-EN/c101202088.lua
Comment (Lyris12) [inline scripts/AGOV-EN/c101202088.lua:1]: Already done: #1191

File: scripts/AGOV-EN/c101202087.lua
Comment (Lyris12) [inline scripts/AGOV-EN/c101202087.lua:1]: Already done: #1191

File: scripts/AGOV-EN/c101202092.lua
Comment (Lyris12) [inline scripts/AGOV-EN/c101202092.lua:1]: Already done: #1192

File: scripts/AGOV-EN/c101202091.lua
Comment (Lyris12) [inline scripts/AGOV-EN/c101202091.lua:1]: Already done: #1192

File: scripts/AGOV-EN/c101202086.lua
Comment (Lyris12) [inline scripts/AGOV-EN/c101202086.lua:1]: Already done: #1192

**Review by Lyris12 — CHANGES_REQUESTED** [review]

File: scripts/AGOV-EN/c101202090.lua
Comment (FluerLee) [reply scripts/AGOV-EN/c101202090.lua:1]: duplicate files have been deleted

**Pull request closed without merging**